packages/client/src/checkout/__fixtures__/patchCheckoutOrderDeliveryBundles.fixtures.ts

```diff
@@ -0,0 +1,17 @@
+import { rest, type RestHandler } from 'msw';
+// import type { Controls } from '../../index.js';
```

Comment:
Check the comments in this file, please.

Reply:
It is here because we were waiting for some information from BE about Controls for this endpoint.
But I just removed it.
Same for other comments

Reply:
Yes, sorry, I hadn't realised at the time that the PR was in draft. Thanks!